### PR TITLE
multiservice example

### PR DESCRIPTION
**Pull request by lsh-0**
I think this serves as a good example but I'm not sure the best place for it to live. Any suggestions, @giorgiosironi 

depends on https://github.com/elifesciences/builder-base-formula/pull/235

### CODE REVIEW

**Comment by giorgiosironi** [comment]
`basebox` is used to generate AMIs that shouldn't contain an empty service. Could use a [different top file](https://github.com/elifesciences/search-formula/blob/master/salt/example-follower-node.top) to build a variant that has it. The alternative is to put it into `heavybox` but it's a longer feedback loop.

**Review by giorgiosironi — APPROVED** [review]

**Comment by giorgiosironi** [comment]
Wait, there's no build here...

**Comment by giorgiosironi** [comment]
The one failure is due to `builder-base-formula` not having the state file yet:
```
[2019-05-16T14:04:54.107Z] [54.82.177.139] out:     No matching sls found for 'elife.multiservice' in env 'base'
```
though I doubt it would work as it will be on 14.04?

**Comment by lsh-0** [comment]
I was thinking heavybox would be a good place for it as this serves as both a demonstration and a test. We'll revisit it once the dust has settled on the main PR

File: salt/basebox/config/lib-systemd-system-test.service [hunk 1]
```diff
@@ -9,4 +9,4 @@ WantedBy={{ process }}-controller.target
 
 [Service]
 RestartSec=10
-ExecStart=/usr/bin/python3 /opt/{{ process }}.py $I
+ExecStart=/usr/bin/python3 /opt/{{ process }}.py %i
```
Comment (lsh-0) [inline salt/basebox/config/lib-systemd-system-test.service:12]: ... was wondering why I wasn't getting that blasted id coming though.

Comment (giorgiosironi) [reply salt/basebox/config/lib-systemd-system-test.service:12]: also with the `onlyif` on the `disable`|`enable` commands, the configuration changes may not have been reloaded while testing so fiddling with the format wouldn't have had an effect until the number of processes was changed and triggered that state.

**Pull request closed without merging**